client/lib/olark/index.js

```diff
@@ -373,3 +377,13 @@ const olark = {
 
 emitter( olark );
 olark.initialize();
+
+export function trackPageViews( context, next ) {
```

Comment:
I'd call this `notifyLocation` or something like that instead. Reading it in `boot` it made me think of stats instead.